src/order/game_add.lean

```diff
@@ -23,29 +23,43 @@ subsequency relation on the addition of combinatorial games.
 
 ## Todo
 
-- Add custom `induction` and `fix` lemmas.
 - Define `sym2.game_add`.
 -/
 
 variables {α β : Type*} (rα : α → α → Prop) (rβ : β → β → Prop)
 
 namespace prod
 
-/-- The "addition of games" relation in combinatorial game theory, on the product type: if
-  `rα a' a` means that `a ⟶ a'` is a valid move in game `α`, and `rβ b' b` means that `b ⟶ b'`
-  is a valid move in game `β`, then `game_add rα rβ` specifies the valid moves in the juxtaposition
-  of `α` and `β`: the player is free to choose one of the games and make a move in it,
-  while leaving the other game unchanged. -/
+/-- `prod.game_add rα rβ x y` means that `x` can be reached from `y` by decreasing either entry. -/
```

Comment:
I don't think removing references to combinatorial games in the docstring is an improvement. `game` is in the name of the relation after all, and provides an important motivation for this operation on relations. Of course your explanation in terms of relations is welcome, so why not combine both?
```suggestion
/-- `prod.game_add rα rβ x y` means that `x` can be reached from `y` by decreasing either entry.
  It is the "addition of games" relation in combinatorial game theory:
  if `rα a' a` means that `a ⟶ a'` is a valid move in game `α`, and `rβ b' b` means that `b ⟶ b'`
  is a valid move in game `β`, then `game_add rα rβ` specifies the valid moves in the juxtaposition
  of `α` and `β`: the player is free to choose one of the games and make a move in it,
  while leaving the other game unchanged. -/
```

Reply:
I'd be fine with copying over the naming rationale from the module docs, but the lengthy paragraph about a specific interpretation of this relation just doesn't seem helpful to me.

Reply:
Sorry for the docstring trouble, both is better.